fulp_modules/features/antagonists/infiltrators/infiltrator.dm

```diff
@@ -53,6 +54,7 @@
 	backpack_contents = list(
 		/obj/item/storage/box/survival/syndie = 1,
 		/obj/item/knife/combat/survival = 1,
+		/obj/item/infiltrator_radio =1,
```

Comment:
```suggestion
		/obj/item/infiltrator_radio = 1,
```

Reply:
Shut the fuck up

Reply:
Who you think you are? You should be kissing my ass right now

Reply:
U mad cuz of my awesome review